person.rb

```diff
@@ -15,6 +15,10 @@ def of_age?
 
   private :of_age?
 
+  def validate_name
+    @name = @corrector.correct_name(@name)
+  end
```

Comment:
- [ ] you are using the `@corrector` instance variable in this method however it was never initialized in the `initialize` method please make sure to initialize this.

Reply:
Thanks for the review @mikethreels